app/javascript/components/time_table/timesheet/entry.js

```diff
@@ -69,6 +70,20 @@ class Entry extends React.Component {
     if (e.key === 'Enter') this.recountTime(this.onSubmit);
   }
 
+  onTimeWheel(e) {
+    e.preventDefault();
+    const { name } = e.target;
+    if (e.deltaY < 0) {
```

Comment:
We could use `Mathi.sign` and omit if statemant, as `moment.add` also works with negative values.

Reply:
Ok, nevermind this function is only supported in firefox.